src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/utils/flowhistory/FlowHistoryRangeConstraints.java

```diff
@@ -46,6 +46,13 @@ public class FlowHistoryRangeConstraints {
     public FlowHistoryRangeConstraints(Optional<Long> timeFromInput,
                                        Optional<Long> timeToInput,
                                        Optional<Integer> maxCountInput) {
+        if (timeFromInput.isPresent() && timeToInput.isPresent() && timeFromInput.get() > timeToInput.get()) {
```

Comment:
I'd move this long condition to private static boolean method with self-explaining name, e.g. `TimeFromIsBeforeTimeTo()`

Reply:
yes, thanks for the comment, will do